elements/pfe-navigation/src/polyfills--pfe-navigation.js

```diff
@@ -0,0 +1,131 @@
+// -- @TODO Needing to manually add polyfills at the moment; check into why babel is not doing this
+
+// -- POLYFILL: Array.prototype.filter
+if (!Array.prototype.filter){
+    Array.prototype.filter = function(func, thisArg) {
+      'use strict';
+      if ( ! ((typeof func === 'Function' || typeof func === 'function') && this) )
+          throw new TypeError();
```

Comment:
same question about curly braces here.

Reply:
@chrisdo1 If this is copied from somewhere, I'd just leave it alone. It would be nice if there was a comment that showed where this code was copied from. Is it from MDN?

Reply:
It's not new with this PR (just moved to a new file) so I don't quite remember but I suspect MDN since that's where I go to find polyfills usually.